### PR TITLE
Added histograms and boxplots for outlier variables

### PR DESCRIPTION
**Pull request by m-visintini**
Please test it and let me know the outcome!
One thing to bear in mind is that the rendering of graphs without outliers only works if the correct ID variable is set up in the Setup tab. This is a bit prone to error at the moment. As soon as I will be done with issue #24 this should not be a problem anymore, since the id variable will be enforced globally.

### CODE REVIEW

**Review by mariarrt94 — CHANGES_REQUESTED** [review]
I really like this feature, especially the **boxplots**. They are super useful for identifying group outliers, and I think they add a lot of value.

### Observations
- For example, when looking at the **`income`** variable grouped, I noticed an **outlier** in `inc_1`. 
- However, when comparing the **histogram with and without outliers**, both look the same. The axis in the "without outliers" view still shows values up to **10,000,000**, which is the identified outlier.
Screenshot 2024-10-24 100939


### Suggestions
- I am unsure whether we should remove these outliers altogether. 
- **Winsorization** might be a great alternative. It could be helpful to show:
  1. The **histogram of `inc_01`**.
  2. The **histogram with winsorized `inc_01`** to see the impact on the distribution.
  3. instead of the filtered_hfc we would create a dataset with the winsorized variable maybe at 95%, let me know if you want to discuss this. 

This is a great addition to the tool—thanks for all the work on it!

**Comment by m-visintini** [comment]
good idea! Will fix.

File: iehfc_app/global.R [hunk 1]
```diff
@@ -7,7 +7,7 @@
   if(!require(pacman)) install.packages("pacman")
 
   pacman::p_load(
-      shiny, dplyr, tidyr, stringr, lubridate, purrr, ggplot2, janitor, data.table, DT, remotes, bsicons,
+      shiny, dplyr, tidyr, stringr, lubridate, purrr, ggplot2, janitor, data.table, DescTools, DT, remotes, bsicons,
```
Comment (m-visintini) [inline iehfc_app/global.R:10]: I actually ended up using a custom function so perhaps we don't need DescTools

**Review by mariarrt94 — COMMENTED** [review]
Great work on this! @m-visintini  I have a few comments:  

1. I changed the name to **winsorized** instead of "without outliers" to be more precise.  
2. I think we should include the level in the title (e.g., **95%**).  I left if as xx%
3. The winsorization should be applied at the **95% level** regardless of the method chosen.  
4. For the boxplots, I believe it’s fine to keep only the with outliers. The only change after winsorization will apply to the selected variable (inc_01), not the entire group (inc), so inc_02, inc_03 are still the same box. The boxplots look clear to me, but I’m happy to discuss this if needed.  

Once these changes are applied, this will be ready to merge!

**Pull request merged**